test/e2e/driver-e2e-specs.js

```diff
@@ -35,4 +35,22 @@ describe('Driver', function () {
     });
     await driver.elementByName('Calculator');
   });
+
```

Comment:
I'd also like to see a test for getClipboard

Reply:
Absolutely - I was just trying to get some advice on what .setClipboard() should behave like before implementing :)